Formula/llvm@9.rb

```diff
@@ -152,6 +203,16 @@ def caveats
   end
 
   test do
+    # Temporary workaround in Big Sur for issues with finding the proper SDK headers.
+    # Described by upstream in https://github.com/llvm/llvm-project/commit/a3a24316087d0e1b4db0b8fee19cdee8b7968032
+    # This fix was backported to LLVM 9 by MacPorts:
+    # https://github.com/macports/macports-ports/blob/master/lang/llvm-9.0/files/patch-clang-fix-include-next-sysroot-cpp-headers.diff
+    # We have copied that to https://raw.githubusercontent.com/Homebrew/formula-patches/1f6960faf59a8d3d83ba8c32d0ec389febfee792/llvm%409/llvm%409.patch
+    # but the patch does not currently fix the include paths.
+    # This workaround should not be needed once that patch works.
+    sdk_include = ""
+    sdk_include = "-isystem #{MacOS.sdk_path}/usr/include" if MacOS.version >= :big_sur
```

Comment:
Big Sur test still failed, unfortunately. I suggest you make this syntax-only and remove this and we can probably merge this and build a bottle for Linux.

Reply:
My thoughts exactly!  As for https://github.com/Homebrew/homebrew-core/pull/80227, I think a rebase should be sufficient after we merge this for the Catalina LLVM bottles to be built.